atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/KeyValueService.java

```diff
@@ -677,4 +677,10 @@ default boolean shouldTriggerCompactions() {
     @DoDelegate
     @Deprecated
     List<byte[]> getRowKeysInRange(TableReference tableRef, byte[] startRow, byte[] endRow, int maxResults);
+
+    @DoDelegate
+    @Override
```

Comment:
Needed to explicitly DoDelegate, so had to override and re-default. Could either just also do true like the AsyncKVS does, or AsyncKVS.super.isValid() which seems jank for this particular case.

isValid is just for shortcircuiting async KVS to avoid lots of throwing and catching exceptions.